pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -58,6 +59,18 @@ func (b *ConsoleServerCLIConfigBuilder) APIServerURL(apiServerURL string) *Conso
 	b.apiServerURL = apiServerURL
 	return b
 }
+func (b *ConsoleServerCLIConfigBuilder) CLIDownloadURL(cliDownloadURL string) *ConsoleServerCLIConfigBuilder {
+	if cliDownloadURL != "" {
+		b.cliDownloadURLs = &CLIDownloadURLs{
```

Comment:
See my other comment about dropping the `DownloadURL` redundancy. I think we can clean this up just a bit.

Reply:
makes sense !